### PR TITLE
fix some php7 type hints

### PR DESCRIPTION
**Pull request by AlaaAttya**

### CODE REVIEW

**Comment by pavinthan** [comment]
@AlaaAttya void support in PHP 7.1, what about php 7.0?

**Comment by TomasVotruba** [comment]
:+1: for adding PHP 7.0 return typehints

**Comment by marky291** [comment]
Can’t we just use the documentation @return, seems pointless unless I’m missing something here.

**Comment by deleugpn** [comment]
Not that I'm in favor of this, but I think Laravel 5.6 will be 7.1+ anyway?

**Comment by mateusjatenee** [comment]
Does Laravel use return types anywhere elae? Unless it was added to the
core recently I don’t remember seeing it anywhere.

File: app/Http/Controllers/Auth/RegisterController.php [hunk 1]
```diff
@@ -45,7 +45,7 @@ public function __construct()
      * @param  array  $data
      * @return \Illuminate\Contracts\Validation\Validator
      */
-    protected function validator(array $data)
+    protected function validator(array $data): \Illuminate\Contracts\Validation\Validator
```
Comment (a-komarev) [inline app/Http/Controllers/Auth/RegisterController.php]: Import it?

File: app/Exceptions/Handler.php [hunk 2]
```diff
@@ -44,9 +44,9 @@ public function report(Exception $exception)
      *
      * @param  \Illuminate\Http\Request  $request
      * @param  \Exception  $exception
-     * @return \Illuminate\Http\Response
+     * @return \Symfony\Component\HttpFoundation\Response
      */
-    public function render($request, Exception $exception)
+    public function render($request, Exception $exception): \Symfony\Component\HttpFoundation\Response
```
Comment (a-komarev) [inline app/Exceptions/Handler.php]: Import it?

**Comment by taylorotwell** [comment]
Not a fan.

**Pull request closed without merging**